build/web_compose/nginx.conf

```diff
@@ -4,10 +4,9 @@ http {
   server {
     listen 8080;
 
-    absolute_redirect off;
-
     location / {
-        proxy_pass http://0.0.0.0:7070/;
+        proxy_pass http://0.0.0.0:7070;
+        proxy_set_header Host $host;
```

Comment:
so the actual fix is "proxy_set_header Host $host;"?

Reply:
Yes.